### ISSUE TITLE
Space in filename of Australian ACT statewide addresses

### ISSUE THREAD

**Issue by orangejulius** [openaddresses/openaddresses#7569]
While doing some processing lately I noticed that there is exactly one address file in all of OA with a space in the filename:

```
julian@Julians-MacBook-Air ~/data/openaddresses $ find . -name '* *'
./au/act/statewide-addresses-ACTGOV Addresses.geojson
```

It's easy to handle spaces in automated scripts, but it's also easy to forget.

I think this space comes from [this line](https://github.com/openaddresses/openaddresses/blob/6996ae0b8509266b8c1e59c1b4c1e551f748cb93/sources/au/act/statewide.json#L13). Can I safely remove the space in a PR to save other people some trouble?

**Comment by iandees** [comment]
Yup, lets change that line. We should probably modify the schema to prevent whitespace in that field too.

**Comment by andrewharvey** [comment]
Sorry that's my fault. I didn't realised it was documented as

> The name of the data provider (AlphaNumeric + Underscores). If unknown, just use the geographic entity, i.e. town, city, county

Agreed though this should be picked up by CI.

**Comment by andrewharvey** [comment]
the specific lines were changed in #7652.
